examples/cargo_without_cmake/src/qml.rs

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+#[cxx::bridge]
+pub mod ffi {
+    unsafe extern "C++" {
+        include!("qml.qrc.cpp");
+        fn qInitResources() -> i32;
+    }
```

Comment:
It would be tricky to automate this, as it needs the CXX part to be within the developers app not in cxx-qt-lib. As it needs to include and link to the qrc file. And this is quite a minimal block that with enough documentation it should be OK.

Maybe later if we can figure out how to use cxx with macros (eg like the `cxx_qt_lib::prelude!()` idea) we could have a `cxx_qt_lib::init_resources!("qrc_a", "qrc_b")` which expands to that. Added this as a note to #287